easybuild/easyblocks/p/parmetis.py

```diff
@@ -167,19 +168,31 @@ def install_step(self):
         # and header files in the Lib directory (capital L). The following symlink are hence created.
         try:
             llibdir = os.path.join(self.installdir, 'Lib')
+            if os.path.lexists(llibdir):
+                os.remove(llibdir)
```

Comment:
please use the `remove_file` and `symlink` functions from `easybuild.tools.filetools`

Reply:
Fixed :+1: